### PR TITLE
Create italian.xml

### PR DESCRIPTION
**Pull request by ecanessa**
## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by mak77** [comment]
which keyboard is this one? it looks different from the usual italian layout (ù position, and missing ,.- keys?)
https://commons.wikimedia.org/wiki/File:Italian_Keyboard_layout.svg

**Comment by hudokkow** [comment]
ping @phate89 for review

File: system/keyboardlayouts/italian.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+Please use English language names instead.
+Default font lacks support for all characters
+-->
+<keyboardlayouts>
+  <layout language="Italian" layout="QWERTY">
+    <keyboard>
+      <row>1234567890'ì</row>
+      <row>qwertyuiopèù</row>
```
Comment (phate89) [inline system/keyboardlayouts/italian.xml]: The ù is in the wrong place..It's in the row under..

File: system/keyboardlayouts/italian.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <row>asdfghjklòà</row>
+      <row>zxcvbnm</row>
```
Comment (phate89) [inline system/keyboardlayouts/italian.xml]: you can add also the 3 characters ,.-

**Review by phate89 — CHANGES_REQUESTED** [review]
Thanks for your contribution..It's ok excluding minor fixes needed

File: system/keyboardlayouts/italian.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<keyboardlayouts>
+  <layout language="Italian" layout="QWERTY">
+    <keyboard>
+      <row>1234567890'ì</row>
+      <row>qwertyuiopè</row>
+      <row>asdfghjklòàù</row>
+      <row>zxcvbnm,.-</row>
+    </keyboard>
+    <keyboard modifiers="shift">
+      <row>1234567890'</row>
```
Comment (Rechi) [inline system/keyboardlayouts/italian.xml]: maybe replace `'` with `?^`

File: system/keyboardlayouts/italian.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      <row>QWERTYUIOPé</row>
+      <row>ASDFGHJKL</row>
```
Comment (Rechi) [inline system/keyboardlayouts/italian.xml]: maybe add `ç°§`

File: system/keyboardlayouts/italian.xml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      <row>ZXCVBNM</row>
```
Comment (Rechi) [inline system/keyboardlayouts/italian.xml]: maybe add `;:_`

**Comment by phate89** [comment]
@emacann also rechi points are good..

**Comment by emacann** [comment]
Yes, I was aware of that but I've had a lot of problems lately. Fixing right now!

**Pull request merged**